client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -452,6 +471,25 @@ const Editor = (props: EditorProps): JSX.Element => {
       null,
       () => {}
     );
+    // Make toggle tab setting in editor permanent
+    // @ts-ignore
+    editor._standaloneKeybindingService.addDynamicKeybinding(
+      '-editor.action.toggleTabFocusMode',
+      null,
+      () => {}
+    );
```

Comment:
```suggestion
```
As far as I can see, this isn't necessary.  The second call with `tabFocusHotkeys` overwrites the existing binding anyways.